theme/sass/components/_blog-sidebar-component.scss

```diff
@@ -0,0 +1,31 @@
+.blog-sidebar-component {
```

Comment:
This file needs to be added into `app.scss`

Reply:
done! So sorry about this.